diff --git a/../work/decompile-f23dd046/net/minecraft/server/EntityShulker.java b/decompile_nms/net/minecraft/server/EntityShulker.java
index 40d2452..e1b633c 100644
--- a/../work/decompile-f23dd046/net/minecraft/server/EntityShulker.java
+++ b/decompile_nms/net/minecraft/server/EntityShulker.java
@@ -65,7 +65,7 @@ public class EntityShulker extends EntityGolem implements IMonster {
     }
 
     public void D() {
-        if (!this.do()) {
+        if (!this.do_()) {
             super.D();
         }
 
@@ -76,7 +76,7 @@ public class EntityShulker extends EntityGolem implements IMonster {
     }
 
     protected SoundEffect bW() {
-        return this.do() ? SoundEffects.fH : SoundEffects.fG;
+        return this.do_() ? SoundEffects.fH : SoundEffects.fG;
     }
 
     protected void i() {
@@ -390,7 +390,7 @@ public class EntityShulker extends EntityGolem implements IMonster {
     }
 
     public boolean damageEntity(DamageSource damagesource, float f) {
-        if (this.do()) {
+        if (this.do_()) {
             Entity entity = damagesource.i();
 
             if (entity instanceof EntityArrow) {
@@ -409,7 +409,7 @@ public class EntityShulker extends EntityGolem implements IMonster {
         }
     }
 
-    private boolean do() {
+    private boolean do_() {
         return this.dj() == 0;
     }
 
diff --git a/../work/decompile-f23dd046/net/minecraft/server/SoundEffectType.java b/decompile_nms/net/minecraft/server/SoundEffectType.java
index 219653c..b106b86 100644
--- a/../work/decompile-f23dd046/net/minecraft/server/SoundEffectType.java
+++ b/decompile_nms/net/minecraft/server/SoundEffectType.java
@@ -11,7 +11,7 @@ public class SoundEffectType {
     public static final SoundEffectType g = new SoundEffectType(1.0F, 1.0F, SoundEffects.ai, SoundEffects.am, SoundEffects.al, SoundEffects.ak, SoundEffects.aj);
     public static final SoundEffectType h = new SoundEffectType(1.0F, 1.0F, SoundEffects.fm, SoundEffects.fq, SoundEffects.fp, SoundEffects.fo, SoundEffects.fn);
     public static final SoundEffectType i = new SoundEffectType(1.0F, 1.0F, SoundEffects.gt, SoundEffects.gx, SoundEffects.gw, SoundEffects.gv, SoundEffects.gu);
-    public static final SoundEffectType j = new SoundEffectType(1.0F, 1.0F, SoundEffects.dl, SoundEffects.dp, SoundEffects.do, SoundEffects.dn, SoundEffects.dm);
+    public static final SoundEffectType j = new SoundEffectType(1.0F, 1.0F, SoundEffects.dl, SoundEffects.dp, SoundEffects.do_, SoundEffects.dn, SoundEffects.dm);
     public static final SoundEffectType k = new SoundEffectType(0.3F, 1.0F, SoundEffects.b, SoundEffects.h, SoundEffects.g, SoundEffects.e, SoundEffects.d);
     public static final SoundEffectType l = new SoundEffectType(1.0F, 1.0F, SoundEffects.fY, SoundEffects.gg, SoundEffects.ge, SoundEffects.gb, SoundEffects.ga);
     public final float m;
diff --git a/../work/decompile-f23dd046/net/minecraft/server/SoundEffects.java b/decompile_nms/net/minecraft/server/SoundEffects.java
index 2cafa4c..6a6fc7b 100644
--- a/../work/decompile-f23dd046/net/minecraft/server/SoundEffects.java
+++ b/decompile_nms/net/minecraft/server/SoundEffects.java
@@ -224,7 +224,7 @@ public class SoundEffects {
     public static final SoundEffect dl;
     public static final SoundEffect dm;
     public static final SoundEffect dn;
-    public static final SoundEffect do;
+    public static final SoundEffect do_;
     public static final SoundEffect dp;
     public static final SoundEffect dq;
     public static final SoundEffect dr;
@@ -475,7 +475,7 @@ public class SoundEffects {
     public static final SoundEffect ic;
     public static final SoundEffect id;
     public static final SoundEffect ie;
-    public static final SoundEffect if;
+    public static final SoundEffect if_;
     public static final SoundEffect ig;
     public static final SoundEffect ih;
     public static final SoundEffect ii;
@@ -732,7 +732,7 @@ public class SoundEffects {
             dl = a("block.ladder.break");
             dm = a("block.ladder.fall");
             dn = a("block.ladder.hit");
-            do = a("block.ladder.place");
+            do_ = a("block.ladder.place");
             dp = a("block.ladder.step");
             dq = a("block.lava.ambient");
             dr = a("block.lava.extinguish");
@@ -983,7 +983,7 @@ public class SoundEffects {
             ic = a("block.wood_pressureplate.click_on");
             id = a("block.wood.step");
             ie = a("entity.zombie.ambient");
-            if = a("entity.zombie.attack_door_wood");
+            if_ = a("entity.zombie.attack_door_wood");
             ig = a("entity.zombie.attack_iron_door");
             ih = a("entity.zombie.break_door_wood");
             ii = a("entity.zombie.death");
